### PR TITLE
Add the KMWP dataset (extension of #3564)

### PR DESCRIPTION
**Pull request by sooftware**
New pull request of #3564 (Add the KMWP dataset)

### CODE REVIEW

File: datasets/kmwp/kmwp.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+class Kmwp(datasets.GeneratorBasedBuilder):
+    """Korean Math Word Problems"""
+
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+
+        features = datasets.Features(
+            {
+                "class": datasets.Value("int32"),
```
Comment (lhoestq) [inline datasets/kmwp/kmwp.py:53]: Can you use ClassLabel here too, as in DKTC ?

**Review by lhoestq — COMMENTED** [review]
Hi ! Here are some comments:

It looks like there are weird characters in the dataset card (in the Data Instances section). Do you think this can be fixed ?

Feel free to also run `make style` to fix the code formatting.

The dummy_data.zip file seems to be empty. Can you re-recreate it ? It must contain a dummy `train.csv` file with only 5 samples inside

File: datasets/kmwp/kmwp.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+"""KMWP (Korean Math Word Problems) Dataset"""
+
+import pandas as pd
+
+import datasets
+
+
+_CITATION = """
+@misc{KMWP
+  author       = {Keum, Bitna and Ryu, Myeonghyeon and Ham, Yoseph and Seo, Minsuh and
```
Comment (lhoestq) [inline datasets/kmwp/kmwp.py:24]: remove trailing whitespace
```suggestion
  author       = {Keum, Bitna and Ryu, Myeonghyeon and Ham, Yoseph and Seo, Minsuh and
```

File: datasets/kmwp/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+licenses:
+- cc-by-nc-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- math-word-problem
```
Comment (lhoestq) [inline datasets/kmwp/README.md:17]:
```suggestion
task_categories:
- conditional-text-generation
task_ids:
- conditional-text-generation-other-math-word-problem
pretty_name: KMWP (Korean Math Word Problems)
```

**Review by lhoestq — COMMENTED** [review]
Thank you :)

The dummy train.csv file seems to be missing one line in order to have one full example. Can you add this missing line to the dummy train.csv file please ?
```
print(y)",18
```

**Comment by sooftware** [comment]
Sorry, I'm late to check! I'll send it to you soon!

**Comment by albertvillanova** [comment]
Thanks for your contribution, @sooftware. Are you still interested in adding this dataset?

We are removing the dataset scripts from this GitHub repo and moving them to the Hugging Face Hub: https://huggingface.co/datasets

We would suggest you create this dataset there, under this organization namespace: https://huggingface.co/tunib

Please, feel free to tell us if you need some help.

**Comment by sooftware** [comment]
Close this PR. Thanks!

**Pull request closed without merging**